apps/helloworld/src/main.js

```diff
@@ -23,5 +25,10 @@ const router = createRouter({
 });
 
 const app = createApp(App);
+
+const pinia = createPinia();
+app.use(pinia);
+useSessionStore(pinia);
+
```

Comment:
useSessionstore not necessary here

Reply:
why not? I mean, it is in the other apps?

Reply:
See comment on other comment, it is necessary (there might be a way to avoid it, but we haven't found it)